source/development.rst

```diff
@@ -0,0 +1,139 @@
+===========
+Development
+===========
+It is recommended that PCDS Python projects are updated and maintained in the
+following way. It is assumed that all PCDS projects are maintained on either
+the SLAC Github or the PCDS specific site. This workflow was not invented here
+at SLAC, there are many helpful tutorials online like `this
+<https://guides.github.com/introduction/flow>`_ if you want more information.
+
+Creating a Local Checkout
+=========================
+If you want to make changes to a repository the first step is to create your
+own fork. This allows you to create feature branches without cluttering the
+main repository. It also assures that the main repository is only added to by
+Pull Request and review. Repositories can be forked from the GitHub site like
+`this <https://help.github.com/articles/fork-a-repo>`_. Once this repository is
+created, you can clone into your own workspace.
+
+.. code:: sh
+
+   $ git clone https://github.com/YOUR-USERNAME/REPOSITORY.git
```

Comment:
Might suggest doing this as `https://YOUR-USERNAME@github.com/YOUR-USERNAME/REPOSITORY.git`, on some git versions this lets you push/pull from origin without putting your git username in every time